myia/ir/visualization/graph_printer.py

```diff
@@ -21,24 +20,29 @@ def __init__(
         graph: Graph,
         *,
         on_node=None,
-        show_constants=True,
+        show_fn_constants=False,
+        show_args=False,
         link_fn_graphs=True,
         link_inp_graphs=True,
+        labeler=None,
     ):
         """Initialize.
 
         :param graph: graph to visualize
         :param on_node: optional function to call on printed node when clicked
-        :param show_constants: if True, display constant non-graph nodes.
-            Constant graphs are always displayed.
+        :param show_fn_constants: if True, display constant nodes when they are used
+            in the FN edge of an Apply node.
```

Comment:
Documentation for parameter `show_args` is missing.

Reply:
True, I'll fix that.

Normally pydocstyle should spot missing parameters, I think the reason it doesn't in this case is that this is a different doc format as the one declared for pydocstyle. I might fix that as well.